webpack/assets/javascripts/foreman_hosts.js

```diff
@@ -0,0 +1,26 @@
+export let pluginEditAttributes = {
```

Comment:
that would imply a state, after registering you are mutating the hosts attributes, and that can cause problems for other consumers.